apps/ui/src/components/App/Topnav.vue

```diff
@@ -56,6 +58,70 @@ const searchConfig = computed(() => {
   return null;
 });
 
+const oldInterfaceLink = computed(() => {
+  const networkSubDomains = {
+    s: 'v1.',
+    's-tn': 'testnet.v1.'
+  };
```

Comment:
We have this that we could use:
https://github.com/snapshot-labs/sx-monorepo/blob/657759e6a7f9f24cd5e6599ef7223f36e71b0108/apps/ui/src/networks/offchain/index.ts#L15-L18